[LOG] Escribiendo logs en: outputs/lstm_summary_line_cls\log\train_20250831-223614.log
Modelo seleccionado: LSTM (Clasificación Summary, *una línea -> clase*)
Modo: SIN ventanas deslizantes (seq_len=1).
Traceback (most recent call last):
  File "C:\Users\adrian\Desktop\TFM\TFG-FR\IA_Meteorologica\Réseaux\Data_base_summaty_class\LSTM\summary_Rainy_Snowy\main_summary_line_cls.py", line 202, in <module>
    main()
  File "C:\Users\adrian\Desktop\TFM\TFG-FR\IA_Meteorologica\Réseaux\Data_base_summaty_class\LSTM\summary_Rainy_Snowy\main_summary_line_cls.py", line 165, in main
    raise ValueError(f"Faltan columnas en el CSV: {missing}")
ValueError: Faltan columnas en el CSV: ['Cloud Cover_clear']
